From f85b2d98cbde4c102bc60654cb2a53f19ec68100 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Tue, 21 Oct 2025 04:25:46 +0200
Subject: [PATCH] Show desktop preferences entry only in LXDE

Most desktop environments have their own desktop icons nowadays,
and PCManFM is used only in LXDE by default.
---
 data/pcmanfm-desktop-pref.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/pcmanfm-desktop-pref.desktop.in b/data/pcmanfm-desktop-pref.desktop.in
index 4f42849d..abbd1b7c 100644
--- a/data/pcmanfm-desktop-pref.desktop.in
+++ b/data/pcmanfm-desktop-pref.desktop.in
@@ -7,4 +7,4 @@ Categories=Settings;GTK;DesktopSettings;X-LXDE-Settings;
 Exec=pcmanfm --desktop-pref
 StartupNotify=true
 Terminal=false
-NotShowIn=GNOME;XFCE;KDE;MATE;
+OnlyShowIn=LXDE;
